frontend/src/taxonomy/transformations.tsx

```diff
@@ -0,0 +1,58 @@
+import { Link } from 'lib/lemon-ui/Link'
+
+type RawCoreFilterDefinition = {
+    label: string
+    description?: string
+    examples?: (string | number)[]
+    system?: boolean
+}
+
+type CoreFilterDefinition = {
+    label: string
+    description?: string | React.ReactNode
+    examples?: (string | number)[]
+    system?: boolean
+}
+
+function transformDescription(description: string): React.ReactNode {
```

Comment:
This is a bit of a sledgehammer approach to generating the markdown. Couldn't quickly get `LemonMarkdown` to do what I wanted.